website/addons/osfstorage/views.py

```diff
@@ -229,3 +229,28 @@ def osfstorage_download(file_node, payload, node_addon, **kwargs):
             osf_storage_settings.WATERBUTLER_RESOURCE: version.location[osf_storage_settings.WATERBUTLER_RESOURCE],
         },
     }
+
+@must_have_permission('write')
+@decorators.autoload_filenode(must_be='file')
+def osfstorage_add_tag(file_node, **kwargs):
```

Comment:
Would it need one? Other methods such as delete do not, and I was under the impression that if a node is a registration the file view methods are not accesible